src/SW_netCDF.c

```diff
@@ -2024,6 +2030,11 @@ void SW_NC_create_progress(SW_DOMAIN* SW_Domain, LOG_INFO* LogInfo) {
             create_full_var(progFileID, NC_BYTE, 0, 0, progVarName,
                             attNames, attVals, numAtts, LogInfo);
         } else {
+            #if defined(SOILWAT)
+            if(LogInfo->printProgressMsg) {
+                sw_message("is creating a progress tracker ...");
```

Comment:
Shouldn't this message (or two similar ones) happen when we create a progress netCDF file and when we add a progress variable to the domain netCDF file?

Reply:
Ah, yes, thanks! -- The location for the progress tracker messages was from before commit 42cc205f02bc4945e061de93de4e6dae01aeed76 "Allow for progress variable to be within domain netCDF". I'll fix this.
`SW_NC_create_domain_template()` contains a progress message for the domain netCDF template -- are you thinking about the need for an additional message?

Reply:
Makes sense, thanks! The only other message I was possibly thinking of, would be something along the lines of `"is adding a progress tracker in the domain file..."` when we need to add the variable to the domain netCDF. And modifying the current on to something like `"is creating a progress tracker in it's own file..."`, to be a bit more descriptive. What do you think?

Reply:
Yes, good idea, though I prefer to keep the use of `sw_message()` as simple as possible and console output as lean as reasonable. Ok with you?

Reply:
I have no problems with this, so it sounds good to me.